### ISSUE TITLE
search with arabic or persian characters

### ISSUE THREAD

**Issue by farenhite** [bjorn2404/jQuery-Store-Locator-Plugin#167]
hi 
when i want to search with arabic or persian characters nothing to show:D

**Comment by bjorn2404** [comment]
@farenhite Yeah, it's a deeper issue with JavaScript. Right now the plugin replaces some of those "foreign" special characters from both the search input and the location data for matching. Last time I looked into it there were some upcoming things with ES6 but I'm not sure if/when I'm going to go that far with this plugin. Looking into possibly doing a Vue, React, or native JS version at some point.
